cli/pack/app_dir.go

```diff
@@ -49,12 +59,33 @@ func initAppDir(appDirPath string, ctx *context.Ctx) error {
 		return err
 	}
 
-	// build
+	cachePaths, err := getProjectCachePaths(ctx)
+	if err != nil {
+		return err
+	}
+
+	if !ctx.Pack.NoCache {
+		if _, err := os.Stat(cachePaths.rocksPath); err == nil {
+			// If rocks found in cache - we just copy them.
+			if err := copyPathFromCache(cachePaths.rocksPath, filepath.Join(appDirPath, ".rocks")); err != nil {
+				return err
+			}
+		} else if !os.IsNotExist(err) {
+			log.Warnf("Failed to copy from cache: %s", err)
+		}
+	}
+
 	ctx.Build.Dir = appDirPath
+	// Build project
 	if err := build.Run(ctx); err != nil {
 		return err
 	}
 
+	// Update rocks cache in cartridge temp directory
+	if err := updateCachePath(ctx, cachePaths, ".rocks"); err != nil {
```

Comment:
Maybe better `updateCache`? `updateCachePath` seems like you updates path to some cache.